okteto.yml

```diff
@@ -0,0 +1,19 @@
+name: java-maven-getting-started
+build:
+  java-maven-getting-started:
+    context: .
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/maven:3
+    command: java -jar *.jar
```

Comment:
`command: mvn spring-boot:run`

Reply:
** bash

Reply:
`mvn spring-boot:run` takes care of installing depedencies, but `npm run start` doesn't. you also need a different command for debugging, which makes a `bash` terminal more convenient for the nodejs sample